src/controllers/authentication.js

```diff
@@ -113,7 +113,15 @@ authenticationController.register = async function (req, res) {
 async function addToApprovalQueue(req, userData) {
 	userData.ip = req.ip;
 	await user.addToApprovalQueue(userData);
-	return { message: '[[register:registration-added-to-queue]]' };
+	let message = '[[register:registration-added-to-queue]]';
+	if (meta.config.showAverageApprovalTime) {
```

Comment:
@oplik0 forgot to mention this, can you add `showAverageApprovalTime` and `autoApproveTime` to `install/data/defaults.json` and set defaults.

Reply:
Sure, I'll add that soon.

Reply:
Hmm... Should the default for `showAverageApprovalTime` be `true` or `false`? I don't really see any harm in enabling it by default for new installations, but I'm not entirely sure about it.

Reply:
Oh well, set it to `true`. I can change it again if `false` is preferred.

Reply:
Yeah default true is fine I think